.github/workflows/e2e-monitoring.yml

```
@@ -0,0 +1,174 @@
+          - performance
+          - accessibility
+
+env:
+  NODE_VERSION: '20'
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+jobs:
+  smoke-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)